src/Model/DashboardInterface.php

```diff
@@ -39,7 +39,7 @@ public function setId($id);
      *
      * @param bool $enabled
```

Comment:
can this be replaced with typehint?

Reply:
You can remove the @param

Reply:
This can be removed if the typehint is added